src/yuzu/configuration/configure_motion_touch.h

```diff
@@ -62,9 +63,12 @@ public slots:
     void ApplyConfiguration();
 
 private slots:
+    void OnUDPAddServer();
+    void OnUDPDeleteServer();
     void OnCemuhookUDPTest();
     void OnConfigureTouchCalibration();
     void OnConfigureTouchFromButton();
+    std::string GetUDPServerString();
```

Comment:
This is a helper function so it shouldn't be alongside the slots.

also:

```suggestion
    std::string GetUDPServerString() const;
```